test/controllers/foods_controller_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class FoodsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @food = foods(:one)
+  end
+
+  test 'should get index' do
+    get foods_url
+    assert_response :success
+  end
+
+  test 'should get new' do
```

Comment:
* I get these errors when i tried to run your tests , kindly see to it and fix the errors . 
![image](https://github.com/AlierPM/Recipedaily/assets/52497246/66bf1417-d379-495e-afec-5036a4424b38)

Reply:
Hello @Davidosky007 ; thank for the constructive review; it is now resolved

Reply:
Hi @AlierPM, your tests are still failing from my end are you available  for a quick call please check your slacl